[Alerting][Docs] Add de-duplication example to ES query docs + update email docs wording (#111026)
elastic/kibana: docs/user/alerting/rule-types/es-query.asciidoc

* An error message is shown if the query is invalid.
+
[role="screenshot"]
image::user/alerting/images/rule-types-es-query-invalid.png[Test {es} query shows error when invalid]

[float]
==== Match de-duplication

> The {es} query rule type performs de-duplication of document matches across rule executions. This means that when the rule is configured with a smaller schedule interval than time window, a document may match a query in multiple rule executions but will be alerted on at most once.

Reviewer: ```suggestion
The {es} query rule type performs de-duplication of document matches across rule executions. If you configure the rule with a schedule interval smaller than the time window, and a document matches a query in multiple rule executions, it will be alerted on only once.
```

> Suppose you have a rule configured to run every minute, using a time window of 1 hour and checking if there are greater than 99 matches for a query. The {es} query rule type will do the following:

Reviewer: ```suggestion
Suppose you have a rule configured to run every minute.  The rule uses a time window of 1 hour and checks if there are more than 99 matches for the query. The {es} query rule type will do the following:
```

Reviewer: Nice table!

[cols="3*<"]
|===

| `Execution 1 (0:00)`
| Rule finds 113 matches in the last hour: `113 > 99`
| Rule is active and user will be alerted.

| `Execution 2 (0:01)`

> | Rule finds 127 matches in the last hour, however, 105 of them are duplicates that were alerted on in Execution 1, so we actually have 22 matches: `22 !> 99`

Reviewer: ```suggestion
| Rule finds 127 matches in the last hour. 105 of the matches are duplicates that were alerted on in Execution 1, so you actually have 22 matches: `22 !> 99`
```

| No alert.

| `Execution 3 (0:02)`

> | Rule finds 159 matches in the last hour, however 88 of them are duplicates that were alerted on in Execution 1, so we actually have 71 matches: `71 !> 99`

Reviewer: ```suggestion
| Rule finds 159 matches in the last hour. 88 of the matches are duplicates that were alerted on in Execution 1, so you actually have 71 matches: `71 !> 99`
```

| No alert.

| `Execution 4 (0:03)`

> | Rule finds 190 matches in the last hour. 71 of them are duplicates that were alerted on in Exeuction 1, so we actually have 119 matches: `119 > 99`

Reviewer: ```suggestion
| Rule finds 190 matches in the last hour. 71 of them are duplicates that were alerted on in Exeuction 1, so you actually have 119 matches: `119 > 99`
```
